asv/www/regressions.js

```diff
@@ -181,13 +181,15 @@ $(document).ready(function() {
                 url_params[key] = [value];
             });
 
-            url_params.time = [];
-            $.each(dates, function(i, date) {
-                if (date[0]) {
-                    url_params.time.push(date[0] + '-' + date[1]);
+            url_params.commits = [];
+            $.each(revisions, function(i, revs) {
+                var commit_b = $.asv.get_commit_hash(revs[1]);
+                if (revs[0]) {
```

Comment:
Yes fixed (I don't known how the revision "0" could be part of a regression, but the code is more clear now).